src/cbmc/CMakeLists.txt

```diff
@@ -1,73 +1,73 @@
 # Library
 file(GLOB_RECURSE sources "*.cpp" "*.h")
 list(REMOVE_ITEM sources
-    ${CMAKE_CURRENT_SOURCE_DIR}/cbmc_main.cpp
-)
+        ${CMAKE_CURRENT_SOURCE_DIR}/cbmc_main.cpp
```

Comment:
Why this line has been tabbed so much?

Similar changes are also everywhere in the file

Reply:
fixed